📜 #14 Docs Updates (#20)
Pan-Canadian-Genome-Library/daco: apps/api/README.md

| `PG_PASSWORD` | Postgres Database password.                                             | `string` | Optional | `mypassword`                                                                                                                          |
| `PG_HOST`                                | URL for the Postgres DB service. application                                                                                             | `string` | Optional | `localhost:5432`                                                                                                                                        |
| `PG_DATABASE`                              | Postgres Database name. application                                                                                           | `string` | Optional | `postgres`                                                                                                                                        |

### Unit Testing

- Unit tests are built using the [Node Test Runner](https://nodejs.org/api/test.html) and are triggered using `pnpm run test`

### Database

> - This application implements a Postgres database using the following [data model](../../docs/model/README.md)

Reviewer: ```suggestion
- This application implements a Postgres database using the following [data model](../../docs/model/README.md)

```

# PCGL DACO API

Nodes.js API for the Pan Canadian Genome Library Data Access Compliance Office.

> This app uses features in Node ^20.9, in combination with TSX for TypeScript support.
> 
> It provides an Express API for accessing a Postgres database using Drizzle ORM.

Reviewer: ```suggestion
This app uses features from Node ^20.9, in combination with TSX for TypeScript support, and it provides access to a Postgres database using Drizzle ORM.
```
